dnf-behave-tests/features/vars-releasever.feature

```diff
@@ -64,3 +64,25 @@ Scenario: Releasever is substituted in baseurl via a value detected from a fedor
     And Transaction is following
         | Action        | Package                       |
         | install       | setup-0:2.12.1-1.fc29.noarch  |
+
+
+@bz1710761
```

Comment:
The scenario should be guarded by @not.with_os=rhel__eq__8 until the fixed dnf is released for RHEL8.